apps/auth0-rules/src/__tests__/add-user-metadata.test.ts

```diff
@@ -77,7 +77,7 @@ const context: RuleContext = {
   authorization: { roles: [] },
 };
 
-const apiUser: UserMetadataResponse = {
+const apiUser: Omit<UserMetadataResponse, 'labs'> = {
```

Comment:
i think `UserMetadataResponse` is a type designed specifically for this use case (ie auth0) and its best to remove the `labs` from that type rather than create something here on the fly